CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 1.3.4 (2020-11-19)
```

Comment:
Please use 1.4.0, as while we do not consider the change breaking, it is still worth a minor version change. Thanks!